plugins/modules/vmware_custom_attribute.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
```

Comment:
How about adding [is_vcenter](https://github.com/ansible-collections/community.vmware/blob/main/plugins/module_utils/vmware.py#L1122) to check whether vCenter because the custom attribute is a feature that vCenter has?